cmds/core/mv/mv_test.go

```diff
@@ -5,278 +5,175 @@
+		},
+	} {
+		*update = true
+		t.Run(tt.name, func(t *testing.T) {
+			if got := move(tt.files); got != nil {
+				if got.Error() != tt.want.Error() {
+					t.Errorf("move() = '%v', want: '%v'", got, tt.want)
+				}
 			}
-		}
+		})
 	}
+
 }
 
-func TestMvUpdate(t *testing.T) {
-	*update = true
-	d, err := setup(t)
+// Test mv func
+func TestMv(t *testing.T) {
+	d, err := setup()
 	if err != nil {
-		t.Error(err)
+		t.Errorf("File setup failed: %v", err)
 	}
 	defer os.RemoveAll(d)
-	t.Logf("Testing mv -u...")
 
-	// Ensure that the newer file actually has a newer timestamp
-	currentTime := time.Now().Local()
-	oldTime := currentTime.Add(-10 * time.Second)
-	err = os.Chtimes(filepath.Join(d, old.n), oldTime, oldTime)
-	if err != nil {
-		t.Error(err)
-	}
-	err = os.Chtimes(filepath.Join(d, new.n), currentTime, currentTime)
-	if err != nil {
-		t.Error(err)
-	}
-
-	// Check that it doesn't downgrade files with -u switch
-	{
-		files := []string{"-u", filepath.Join(d, old.n), filepath.Join(d, new.n)}
-		res := testutil.Command(t, files...)
-		_, err = res.CombinedOutput()
-		if err = testutil.IsExitCode(err, 0); err != nil {
-			t.Error(err)
-		}
-		newContent, err := os.ReadFile(filepath.Join(d, new.n))
-		if err != nil {
-			t.Error(err)
-		}
-		if bytes.Equal(newContent, old.c) {
-			t.Error("Newer file was overwritten by older file. Should not happen with -u.")
-		}
-	}
+	for _, tt := range []struct {
+		name  string
+		files []string
+		want  error
+	}{
+		{
+			name:  "len(files) > 2",
+			files: []string{filepath.Join(d, "hi1.txt"), filepath.Join(d, "hi2.txt"), d},
+			want:  fmt.Errorf(""),
+		},
+		{
+			name:  "len(files) > 2 && d does not exist",
+			files: []string{filepath.Join(d, "hi1.txt"), filepath.Join(d, "hi2.txt"), "d"},
+			want:  fmt.Errorf("lstat %s: no such file or directory", filepath.Join("d", "hi1.txt")),
+		},
+		{
+			name:  "len(files) = 2",
+			files: []string{filepath.Join(d, "hi1.txt"), filepath.Join(d, "hi2.txt")},
+			want:  fmt.Errorf(""),
+		},
+	} {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := mv(tt.files, false); got != nil {
+				if got.Error() != tt.want.Error() {
+					t.Errorf("mv() = '%v', want: '%v'", got, tt.want)
+				}
+			}
 
-	// Check that it does update files with -u switch
-	{
-		files := []string{"-u", filepath.Join(d, new.n), filepath.Join(d, old.n)}
-		res := testutil.Command(t, files...)
-		_, err = res.CombinedOutput()
-		if err = testutil.IsExitCode(err, 0); err != nil {
-			t.Error(err)
-		}
-		newContent, err := os.ReadFile(filepath.Join(d, old.n))
-		if err != nil {
-			t.Error(err)
-		}
-		if !bytes.Equal(newContent, new.c) {
-			t.Error("Older file was not overwritten by newer file. Should happen with -u.")
-		}
-		if _, err := os.Lstat(filepath.Join(d, old.n)); err != nil {
-			t.Error("The new file shouldn't be there anymore.")
-		}
+		})
 	}
 }
 
-func TestMvNoClobber(t *testing.T) {
-	*noClobber = true
-	d, err := setup(t)
+// Test moveFile func
```

Comment:
I bet some go tool will complain about this comment. 
Change to 
// TestMoveFile tests that the blarg does the foo without problems. 

Or some useful comment. Or remove the comment entirely, I'm not sure it's needed.